l10n_it_ricevute_bancarie/views/account_view.xml

```diff
@@ -115,13 +115,27 @@
         <field name="model">account.invoice</field>
         <field name="inherit_id" ref="account.invoice_form"/>
         <field name="arch" type="xml">
+            <xpath expr="//field[@name='payment_term_id']" position="after">
+                <field name="is_unsolved" string="Unsolved"
```

Comment:
L'attributo `string` non è necessario, ma per me si può mergiare lo stesso.

Reply:
> L'attributo `string` non è necessario, ma per me si può mergiare lo stesso.

@SimoneVagile Se mantieni la stringa il testo andrebbe corretto con "Past Due", perché unsolved è un "false friend" in questo contesto.

Vedi https://github.com/tafaRU/l10n-italy/pull/8

Reply:
@primes2h grazie per la review.